src/app/vault/vault.component.ts

```diff
@@ -660,6 +660,7 @@ export class VaultComponent implements OnInit, OnDestroy {
             this.platformUtilsService.copyToClipboard(value);
             this.toasterService.popAsync('info', null,
                 this.i18nService.t('valueCopied', this.i18nService.t(labelI18nKey)));
+            this.viewComponent.minimizeIfNeeded();
```

Comment:
Does this function get called in other contexts outside of viewComponent? Shouldn't you only do this if `this.viewComponent != null && this.action === 'view'`?

Reply:
Good point. The null check is needed. However, the user might be editing a cipher and while editing, copy a username from another cipher. If the window wouldn't minimize, that's not consistent with the minimize option turned on, which is what the user expects. I would suggest to move the minimizeIfNeeded() method somewhere else.